packages/orbit-components/src/Itinerary/README.md

```diff
@@ -70,8 +70,8 @@ ItinerarySegmentStop is an atomic unit of the Itinerary component, shows two loc
 
 | Name             | Type                | Required           | Default       | Description                                              |
 | ---------------- | ------------------- | ------------------ | ------------- | -------------------------------------------------------- |
-| date             | `React.Node`        | :heavy_check_mark: |               | The date of `ItinerarySegmentStop`                       |
-| time             | `React.Node`        | :heavy_check_mark: |               | The time of `ItinerarySegmentStop`                       |
+| date             | `React.Node`        |                    |               | The date of `ItinerarySegmentStop`                       |
```

Comment:
Can you also remove the empty row in this table? It was not caused by your changes, but still can be done here ;)

Reply:
What empty row? 🤔